docs/using-keystone-webhook-authenticator-and-authorizer.md

```diff
@@ -16,28 +20,35 @@ kubelet) or a normal kubernetes service.
 
 ## Running k8s-keystone-auth as a Kubernetes service
 
+First, create a folder in which we will put all the manifest files and
```

Comment:
First, create a folder to store all the manifest files and config files.

Reply:
Why? I don't see there is much difference.